grails-app/views/inventory/listExpiredStock.gsp

```diff
@@ -71,6 +71,7 @@
                             </h2>
 		          			<div class="filter-list-item">
 		           				<label><warehouse:message code="category.label"/></label>
+		           				%{-- not sure what the ${{...}} in optionValue means: it may need to be de-escaped. OBPIH-5506 --}%
```

Comment:
I added this comment because I have no idea what is going on in line 77. What does `${{...}}` do here? I either need to test it or read the spec or ask a reviewer.

Reply:
@mdpearson you can use **closures** if you need to make some additional operations on `optionValue` instead of just using e.g. `optionValue="name"`

we can write for example:
```groovy
 optionValue="${{ it?.name?.toUpperCase() }}"
```

Reply:
@kchelstowski thanks, that's very helpful!